torchrl/objectives/ppo.py

```diff
@@ -71,6 +73,19 @@ class PPOLoss(LossModule):
             policy and critic will only be trained on the policy loss.
             Defaults to ``False``, ie. gradients are propagated to shared
             parameters for both policy and critic losses.
+        kl_init_coef (scalar, optional): A coefficient for the KL-divergence
```

Comment:
`kl_init_coef` let think that this is an initial value for kl. what about simply `kl_coef`?

Reply:
We have other KL coefs we need to distinguish them no?

Reply:
we need a name that says its the kl vs the initial/reference model. `initial_model_kl_coeff`? its long but I can't think to anything better right now

also what about passing a "config" (aka a dict/dataclass) instead? this enables to implement advanced kl controllers like [here](https://github.com/lvwerra/trl/blob/25fa1bd8803c9998d95a3c38b2bb27a85235ce7b/trl/trainer/ppo_trainer.py#L269)

Reply:
I changed it to kl_coef, I think it's fine. the other one is named beta and it is used differently. There's no risk of clashing IMO